.appveyor.yml

```diff
@@ -0,0 +1,81 @@
+image: Visual Studio 2019
+platform:
+   - amd64
+
+environment:
+   APPVEYOR_API_TOKEN:
+      secure: qTt+0485sfAXM2Sz+QLZ+jjziNf4One8/A062QFN1a4=
+   GITHUB_API_TOKEN:
+      secure: TpVl9vErdKkXR/RUlVtUDfeITAn5BtOhG3JxiFJRGp85c3B7boCFf6CDOxNgMr/m
+   DEPLOY_BRANCH: 'master'
```

Comment:
I think these are encrypted based on a particular appveyor account - https://www.appveyor.com/docs/build-configuration/#secure-variables

I assume these values would need to be updated once we decide what accounts will be used?

Reply:
Correct. For github the key needs `repo` permissions. I can walk you trough creating and setting everything up once we're there